ex1.py

```diff
@@ -2,11 +2,8 @@
 import math
 
 def area(r):
-    return((math.pi*r**2))
+    return math.pi*r**2 
 
 if __name__ == "__main__":
  r = float(input("please enter a value: "))
  print(f"The answer is:{area(r)}")
```

Comment:
the spaces made for lines 8 and 9 are not as  rerquired, you used only one space. you should use a tab (or 4 regular spaces) to create a scope. as you did in function "area" in line 4,5:
line 4 has no spaces at the beginning but line 5 has a tab space.